packages/frontend/src/pages/dashboard.md

```diff
@@ -10,6 +10,10 @@ AI Studio ships with a so-called Recipes Catalog that helps you navigate a numbe
 
 AI Studio provides a curated list of open source AI models and LLMs. Once downloaded, the models are available to be used for AI applications, model services and playgrounds.
 
+## Model serving
+
+Once a model has been downloaded, users can start an inference server for the model. This allow to test the model using a playground environment or to connect applications as the inference server is exposing a well know chat API.
```

Comment:
```suggestion
Once a model has been downloaded, you can start an inference server for the model. This allows to test the model using a playground environment or to connect applications as the inference server is exposing a well know chat API.
```

Can we mention which API is being used?  Maybe adding that it's an open-source standard (in case it is)?

Reply:
That was on purpose as the API in question is the OpenAI chat completion API (a de facto standard)